Backend/dev/migrations/rider-app/1171-add-customer-cancellation-config.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE atlas_app.merchant ADD column cancellation_fee int DEFAULT 10 NOT NULL;
+ALTER TABLE atlas_app.merchant ADD column driver_pickup_distance double precision DEFAULT 0 NOT NULL;
+ALTER TABLE atlas_app.merchant ADD column driver_booking_time int DEFAULT 10 NOT NULL;
+ALTER TABLE atlas_app.merchant ADD column dispute_limit int DEFAULT 3 NOT NULL;
+ALTER TABLE atlas_app.merchant ADD column ride_cancellation_limit int DEFAULT 3 NOT NULL;
```

Comment:
Also, we should block the customer if they cross this threshold, right? I didn't see that logic in this PR

Reply:
Nope we should not otherwise he will never be able to pay the dues as we are adding the cancellationDues in endRide fare